code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -469,3 +469,27 @@
 	desc = "For when you're stepping on up to the plate."
 	icon_state = "JackFrostShoes"
 	inhand_icon_state = "JackFrostShoes_worn"
+
+/obj/item/clothing/shoes/phantom
+	name = "Phantom Shoes"
+	desc = "Excellent for when you need to do cool flashy flips."
+	icon_state = "phantom_shoes"
+	inhand_icon_state = "phantom_shoes"
+
+/obj/item/clothing/shoes/saints
+	name = "Saints sneakers"
+	desc = "Officially branded Saints sneakers. Incredibly valuable!"
+	icon_state = "saints_shoes"
+	inhand_icon_state = "saints_shoes"
+
+/obj/item/clothing/shoes/morningstar
+	name = "Morningstar boots"
+	desc = "The most expensive boots on this station. Wearing them dropped the value by about 50%."
```

Comment:
```suggestion
	desc = "The most expensive boots on this station. Wearing them dropped their value by about 50%."
```

Reply:
actually, suggestion: only add the "Wearing them dropped their value by about 50%." bit of the description after you put them on